tools/functional-tester/etcd-tester/checks.go

```diff
@@ -87,29 +89,93 @@ func (lc *leaseChecker) Check() error {
 		return err
 	}
 	plog.Infof("checking short lived leases %v", lc.ls.shortLivedLeases.leases)
-	return lc.check(true, lc.ls.shortLivedLeases.leases)
+	return lc.shortLivedLeaseCheck()
+
+}
+
+// shortLivedLeaseCheck verifies the expiry of shortLivedLeases.
+// the logic is follwing:
```

Comment:
Why isn't `checkShortLivedLeases ensures leases expire.` enough? Listing what the function does like this is repeating what's already in there code-- are there edge cases that wouldn't be obvious from reading the code?